recipes/UGrid/meta.yaml

```
@@ -0,0 +1,49 @@
+{% set name = "UGrid" %}
+{% set version = "0.0.1.dev" %}
+{% set commit = "79841fe3af2cf60eece683e6cff2d5dc31690698" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/Deltares/UGrid/archive/{{ commit }}.zip
+  sha256: a8c42f8d23c947de342dcb3a82102487c9af207a3e10da0d0228b8e9c1df7575
+  patches:
+    - patches/constans.patch
+    - patches/use_runtime_error.patch
+
+build:
+  skip: True  # [win]
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - cmake
+    - make
+  host:
+    - boost
+    - boost-cpp
+    - netcdf-cxx4
```

you might need limb here as well
```suggestion
    - netcdf-cxx4
    - openlibm
```